ui/search/nuxeo-search-form-layout.js

```diff
@@ -95,7 +96,12 @@ import { RoutingBehavior } from '../nuxeo-routing-behavior.js';
         /**
          * The base url for the layout href.
          */
-        hrefBase: String,
+        hrefBase: {
+          type: String,
+          value() {
+            return config.get('searchForm.hrefBase');
```

Comment:
I guess we should name this after the element, so it could be instead:
```suggestion
            return config.get('searchFormLayout.hrefBase');
```

Reply:
The naming makes sense, keeps it consistent.